### PR TITLE
BEDtools merge v2.28.0_cv2

### PR DESCRIPTION
**Pull request by shalabhsuman**
#42

### CODE REVIEW

File: bedtools_merge_v2.28.0_cv2/bedtools_merge.cwl [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    inputBinding:
+      position: 0
+      prefix: '-d'
+      shellQuote: false
+    doc: Maximum distance between features allowed for features to be merged.
+outputs:
+  - id: output_file
+    type: File?
+    outputBinding:
+      glob: |2-
```
Comment (rhshah) [inline bedtools_merge_v2.28.0_cv2/bedtools_merge.cwl]: how is specifying `|-` different than `|2-`❓

File: bedtools_merge_v2.28.0_cv2/bedtools_merge.cwl [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+          ${
+            if (inputs.output_file_name.length>0)
```
Comment (rhshah) [inline bedtools_merge_v2.28.0_cv2/bedtools_merge.cwl]: Why are we looking at the length of the file❓

File: bedtools_merge_v2.28.0_cv2/bedtools_merge.cwl [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+label: bedtools_merge
+requirements:
+  - class: ShellCommandRequirement
+  - class: ResourceRequirement
+    ramMin: "${\r  if(inputs.memory_per_job && inputs.memory_overhead) {\r   \r    return inputs.memory_per_job + inputs.memory_overhead\r  }\r  else if (inputs.memory_per_job && !inputs.memory_overhead){\r    \r   \treturn inputs.memory_per_job + 2000\r  }\r  else if(!inputs.memory_per_job && inputs.memory_overhead){\r    \r    return 8000 + inputs.memory_overhead\r  }\r  else {\r    \r  \treturn 8000 \r  }\r}"
+    coresMin: "${\r  if (inputs.number_of_threads) {\r    \r   \treturn inputs.number_of_threads \r  }\r  else {\r    \r    return 1\r  }\r}"
+  - class: DockerRequirement
+    dockerPull: 'biocontainers/bedtools:v2.28.0_cv2'
+  - class: InlineJavascriptRequirement
+stdout: |2-
```
Comment (rhshah) [inline bedtools_merge_v2.28.0_cv2/bedtools_merge.cwl]: Same question as above ❔

File: bedtools_merge_v2.28.0_cv2/bedtools_merge.cwl (hunk 2 above)
Comment (shalabhsuman) [reply bedtools_merge_v2.28.0_cv2/bedtools_merge.cwl]: to check for an **empty** string in the input YAML

File: bedtools_merge_v2.28.0_cv2/bedtools_merge.cwl (hunk 3 above)
Comment (shalabhsuman) [reply bedtools_merge_v2.28.0_cv2/bedtools_merge.cwl]: it was from Rabix whenever it sees a `" "` in the java script. I resolved it.

File: bedtools_merge_v2.28.0_cv2/bedtools_merge.cwl (hunk 1 above)
Comment (shalabhsuman) [reply bedtools_merge_v2.28.0_cv2/bedtools_merge.cwl]: it was from Rabix whenever it sees a `" "` in the java script. I resolved it.

**Pull request merged**